README.md

```diff
@@ -85,20 +85,33 @@ Install and uninstall `DBLINK()` with the repository [Makefile](Makefile).
 - Specific ODBC Drivers for the remote database that you want to connect to
 - [Configure the ODBC layer](#configure-the-odbc-layer)
 
-### Install DBLINK()
+### Installing pre-built binaries
 
-> Before you run `make` commands, review the [Makefile](Makefile) and make any necessary changes.
+1. Download the appropriate build of ldblink.so from the [latest release](/vertica/dblink/releases) to `ldblink.so`.
+2. Copy `ldblink.so` to the initiator node
```

Comment:
Period after 2. node.
colon after 3. ...initiator node:
period after 4. node.

I would put (optional) at the beginning of the step:

4. (Optional) Delete ldblink....